[2021-03-01 20:43:44,866] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.storing_user 2020-03-18T00:00:00+00:00 [queued]>
[2021-03-01 20:43:44,874] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.storing_user 2020-03-18T00:00:00+00:00 [queued]>
[2021-03-01 20:43:44,874] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:43:44,874] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-01 20:43:44,874] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:43:44,883] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): storing_user> on 2020-03-18T00:00:00+00:00
[2021-03-01 20:43:44,886] {standard_task_runner.py:51} INFO - Started process 245045 to run task
[2021-03-01 20:43:44,889] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'storing_user', '2020-03-18T00:00:00+00:00', '--job-id', '447', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpvv1chgvk']
[2021-03-01 20:43:44,889] {standard_task_runner.py:76} INFO - Job 447: Subtask storing_user
[2021-03-01 20:43:44,912] {logging_mixin.py:103} INFO - Running <TaskInstance: user_processing.storing_user 2020-03-18T00:00:00+00:00 [running]> on host pop-os.localdomain
[2021-03-01 20:43:44,938] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=storing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-03-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-18T00:00:00+00:00
[2021-03-01 20:43:44,938] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-01 20:43:44,939] {bash.py:158} INFO - Running command: echo -e ".separator ","
.import /tmp/processed_user.csv users" | sqlite3 /home/i-sip_iot/airflow/airflow.db
[2021-03-01 20:43:44,945] {bash.py:169} INFO - Output:
[2021-03-01 20:43:44,961] {bash.py:177} INFO - Command exited with return code 0
[2021-03-01 20:43:44,982] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=storing_user, execution_date=20200318T000000, start_date=20210302T014344, end_date=20210302T014344
[2021-03-01 20:43:44,999] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-01 20:43:45,020] {local_task_job.py:118} INFO - Task exited with return code 0
